File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int j = s.size()-1;
        while(s[j] == ' ') j--;
        string a;

        while(j >= 0){
            if(s[j] == ' ') return a.size();
            a += s[j]; j--;
        }
        return a.size();
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String sb=s.trim();
        int j=sb.length()-1;
        int count=0;
        while(j>=0 && sb.charAt(j)!=' '){
            count++;
            j--;
        }
        return count;
    }
}